MODULE.bazel

```diff
@@ -30,27 +31,37 @@ bazel_dep(name = "rules_rust", version = "0.49.1")
 
 bazel_dep(name = "buildifier_prebuilt", version = "6.4.0", dev_dependency = True)
 
-crate = use_extension(
+# crate_py but shortened due to Windows file path considerations
+cp = use_extension(
```

Comment:
really minor cosmetic nit, but is there any reason to use `cp` here and just `r` for ruby? If `cr` for ruby is already too much, maybe then also use `p` here?

Reply:
Yeah, I wanted to limit the use of single-letter repo names to where we really need them (as there's only 26 of them available). The more letters, the easier to search for the repo name in our source code, too.